sdk/aqueduct/dag.py

```diff
@@ -50,10 +52,23 @@ class AirflowEngineConfig(BaseModel):
     integration_id: uuid.UUID
 
 
+class K8sEngineConfig(BaseModel):
+    integration_id: uuid.UUID
+
+
 class EngineConfig(BaseModel):
     type: RuntimeType = RuntimeType.AQUEDUCT
```

Comment:
Could you add a quick note here that the runtime type dictates which engine config we expect to be set? The relationship between fields might be non-obvious to readers since we use different terms "runtime" and "engine" .